docs/gaia/installation.md

```diff
@@ -40,6 +40,37 @@ $ gaiad version
 $ gaiacli version
 ```
 
+### Install binary distribution via snap (Linux only)
+
+Gaia can be installed on various GNU/Linux distributions from the [Snapcraft.io](https://snapcraft.io/gaia) store:
+
+```bash
+$ sudo snap install gaia
+```
+
+Development builds are available through the `edge` channel:
+
+```bash
+$ sudo snap install --edge gaia
+```
+
+::: tip
+At the time of writing, only the following [architectures are supported](https://build.snapcraft.io/user/cosmos/cosmos-sdk): `amd64` `i386` `arm64` `armhf` `ppc64el` `s390x`.
+:::
+
+`snap` installs Gaia binaries as `gaia.gaiad` and `gaia.gaiacli`. It is recommended to create commands aliases for user's convenience once the package is installed:
+
+```
+$ sudo snap alias gaia.gaiad gaiad
+$ sudo snap alias gaia.gaiacli gaiacli
+```
+
+::: warning
+Note that the binaries provided by the snap package save their data into **$HOME/snap/gaia/** instead of **$HOME**.
```

Comment:
Any way to change this?

Reply:
(can we test with `--home`)

Reply:
Tested, it works as long as users do not try to forcibly write to `~/.gaia{cli,d}`. It simply isn't allowed by security confinement. Still, they can write other dirs in their `$HOME`:

```
$ gaia.gaiacli keys add --home=/home/alessio/gaiacli foo
Enter a passphrase to encrypt your key to disk:
Repeat the passphrase:

NAME:	TYPE:	ADDRESS:						PUBKEY:
foo	local	cosmos1dp5jptpxhuws32yxjzqe70v64akxrpsws2cpgz	cosmospub1addwnpepqgdud3pdh6ckym8l09hc6jsyaxeefpkdhfxxh6746f6d5s0a9tf2c4phfnu

**Important** write this mnemonic phrase in a safe place.
It is the only way to recover your account if you ever forget your password.

silver pulse oak dismiss pencil prevent symbol display raccoon inflict favorite allow prevent double unhappy dream bracket old green maid aware minute dust inside
```

Reply:
Why not those directories? Are directories starting with `.` prohibited?

Reply:
We should probably document this.

Reply:
Exactly, dotfiles and directories cannot be accessed: https://forum.snapcraft.io/t/syncthing-cannot-access-hidden-files-and-directories/745

Reply:
We should def document this. With some warnings.

Reply:
WIll do in a separate PR